src/services/test-helpers/mock/mockAssetHubKusamaApiBlock3356195.ts

```diff
@@ -0,0 +1,296 @@
+	weight: {
+		refTime: 1200000000,
+		proofSize: 20000,
+	},
+	class: 'Normal',
+	partialFee: 149000000,
+});
+
+export const assetHubKusamaQueryInfoCall3356195 = (
+	_extrinsic: GenericExtrinsic,
+	_length: Uint8Array,
+): Promise<RuntimeDispatchInfoV2> => Promise.resolve().then(() => runtimeDispatchInfoV2);
+
+export const assetHubKusamaQueryInfoAt3356195 = (_extrinsic: string, _hash: Hash): Promise<RuntimeDispatchInfoV2> =>
+	Promise.resolve().then(() => runtimeDispatchInfoV2);
+
+export const assetHubKusamaSubmitExtrinsic3356195 = (_extrinsic: string): Promise<Hash> =>
+	Promise.resolve().then(() => assetHubKusamaRegistryV1000000.createType('Hash'));
+
+const getStorage = () => Promise.resolve().then(() => assetHubKusamaRegistryV1000000.createType('Option<Raw>', '0x00'));
+
+const chainType = () =>
+	Promise.resolve().then(() =>
+		assetHubKusamaRegistryV1000000.createType('ChainType', {
+			Live: null,
+		}),
+	);
+
+const properties = () =>
+	Promise.resolve().then(() =>
+		assetHubKusamaRegistryV1000000.createType('ChainProperties', {
+			ss58Format: '2',
+			tokenDecimals: '12',
+			tokenSymbol: 'KSM',
+		}),
+	);
+
+const getFinalizedHead = () => Promise.resolve().then(() => blockHash3356195);
+
+const health = () =>
+	Promise.resolve().then(() => assetHubKusamaRegistryV1000000.createType('Health', '0x7a000000000000000001'));
+
+const localListenAddresses = () =>
+	Promise.resolve().then(() => assetHubKusamaRegistryV1000000.createType('Vec<Text>', localListenAddressesHex));
+
+const nodeRoles = () =>
+	Promise.resolve().then(() => assetHubKusamaRegistryV1000000.createType('Vec<NodeRole>', '0x0400'));
+
+const localPeerId = () =>
+	Promise.resolve().then(() =>
+		assetHubKusamaRegistryV1000000.createType('Text', '12D3KooWA88bs6zSQmvvTvwqgW8RDFos7KsJrfCAXjm9QyDESgtU'),
+	);
+
+export const assetHubKusamaPendingExtrinsics3356195 = (): Promise<Vec<Extrinsic>> =>
+	Promise.resolve().then(() => assetHubKusamaRegistryV1000000.createType('Vec<Extrinsic>'));
+
+export const assetHubKusamaTx3356195 = (): Extrinsic =>
+	assetHubKusamaRegistryV1000000.createType('Extrinsic', balancesTransferValid);
+
+const traceBlock = () =>
+	Promise.resolve().then(() => assetHubKusamaRegistryV1000000.createType('TraceBlockResponse', traceBlockRPC.result));
+
+/**
+ * Deafult Mock polkadot-js ApiPromise. Values are largely meant to be accurate for block
+ * #3356195, which is what most Service unit tests are based on.
+ */
+export const mockAssetHubKusamaApiBlock3356195 = {
+	runtimeVersion,
+	call: {
+		transactionPaymentApi: {
+			queryInfo: assetHubKusamaQueryInfoCall3356195,
+			queryFeeDetails,
+		},
+	},
+	consts: {
+		system: {
+			blockLength: {
+				max: {
+					normal: new BN(3932160),
+					operational: new BN(5242880),
+					mandatory: new BN(5242880),
+				},
+			},
+			blockWeights: {
+				baseBlock: new BN(5481991000),
+				maxBlock: assetHubKusamaRegistryV1000000.createType('u64', 15),
+				perClass: {
```

Comment:
Looks like `perClass` can be reused among the new mockApi's

Reply:
True, very good point! 💯 I reused mockApi's `perClass` in all the rest mocked apis.